src/service-override/secretStorage.ts

```diff
@@ -0,0 +1,13 @@
+import { IEditorOverrideServices } from 'vs/editor/standalone/browser/standaloneServices'
+import { IEncryptionService } from 'vs/platform/encryption/common/encryptionService'
+import { SyncDescriptor } from 'vs/platform/instantiation/common/descriptors'
+import { ISecretStorageService } from 'vs/platform/secrets/common/secrets'
+import { EncryptionService } from 'vs/workbench/services/encryption/browser/encryptionService'
+import { BrowserSecretStorageService } from 'vs/workbench/services/secrets/browser/secretStorageService'
+
+export default function getServiceOverride (): IEditorOverrideServices {
+  return {
+    [ISecretStorageService.toString()]: new SyncDescriptor(BrowserSecretStorageService, [], true),
+    [IEncryptionService.toString()]: new SyncDescriptor(EncryptionService, [], true)
+  }
+}
```

Comment:
What about re-exporting the `LocalStorageSecretStorageProvider` from VSCode?

Reply:
Good idea! Will do that.

Reply:
Okay, hmm, so it looks like the `LocalStorageSecretStorageProvider` is not part of the https://github.com/microsoft/vscode/blob/main/src/vs/code/browser/workbench/workbench.ts#L168C14-L168C47). I can't seem to import it here, because it's not included in the VSCode sources we install. I _can_ copy over the code and use that though, what do you think?

Reply:
we're just removing the whole vs/code directory in the install-vscode script, we can probably make it more specific

Reply:
It wasn't a good idea, because it makes us pull the whole workbench... and it seems that provider rely on a dom element existing anyway, so It's maybe too specific